### ISSUE TITLE
getImage() throws ArgumentError (Invalid argument: Instance of 'ImageDimension')

### ISSUE THREAD

**Issue by kalkih** [brim-borium/spotify_sdk#78]
**Describe the bug**
`getImage()` throws exception `ArgumentError (Invalid argument: Instance of 'ImageDimension')`.
Tried with and without passing in the `dimension` parameter. Also tried with different ImageDimensions, no luck.

Other api methods work fine.

Spotify SDK versions:
- **Spotify app** remote: spotify-app-remote-release-0.7.1.aar
- **Spotify auth**: spotify-auth-release-1.2.3.aar

Stacktrace:
```
E/flutter (31567): [ERROR:flutter/lib/ui/ui_dart_state.cc(186)] Unhandled Exception: Invalid argument: Instance of 'ImageDimension'
E/flutter (31567): #0      StandardMessageCodec.writeValue
package:flutter/…/services/message_codecs.dart:403
E/flutter (31567): #1      StandardMessageCodec.writeValue.<anonymous closure>
package:flutter/…/services/message_codecs.dart:400
E/flutter (31567): #2      _LinkedHashMapMixin.forEach (dart:collection-patch/compact_hash.dart:397:8)
E/flutter (31567): #3      StandardMessageCodec.writeValue
package:flutter/…/services/message_codecs.dart:398
E/flutter (31567): #4      StandardMethodCodec.encodeMethodCall
package:flutter/…/services/message_codecs.dart:535
E/flutter (31567): #5      MethodChannel._invokeMethod
package:flutter/…/services/platform_channel.dart:150
E/flutter (31567): #6      MethodChannel.invokeMethod
package:flutter/…/services/platform_channel.dart:331
E/flutter (31567): #7      SpotifySdk.getImage
package:spotify_sdk/spotify_sdk.dart:501

```
**To Reproduce**
1. Establish spotify remote connection
2. Call the following code
  ```dart
void getImage() async {
    print('GETTING IMAGE');
    try {
      var image = await SpotifySdk.getImage(
        imageUri:
            ImageUri('spotify:image:ab67616d0000b2736b4f6358fbf795b568e7952d'),
        dimension: ImageDimension.large,
      );
      print(image);
    } on PlatformException catch (error) {
      print(error);
    } on MissingPluginException catch (error) {
      print(error);
    }
  }
  ```
3. See exception thrown

**Expected behavior**
Should return image without throwing exception.

**Screenshots**
<img width="841" alt="Screenshot 2021-03-07 at 12 22 41" src="https://user-images.githubusercontent.com/457678/110238233-d1d58d00-7f40-11eb-861c-f17c25a39004.png">

**Smartphone (please complete the following information):**
 - Device: Pixel 3 (Emulator)
 - OS: Android 11 (API 30)
 - Version (spotify_sdk): 1.0.1

**Comment by brim-borium** [comment]
Did this work for you with the previous version of the package? I put the ImageDimensions in their own class, seemed to work for the example file.

**Comment by kalkih** [comment]
I just recently started using this package so **1.0.1** is the first version I've tried.
Great package btw, thanks for your work!

Just tried downgrading to version **1.0.0** and `getImage()` is working as expected on that version.

**Comment by brim-borium** [comment]
In your file where you use this are you importing the image dimension enum like this ? 

`import 'package:spotify_sdk/enums/image_dimension_enum.dart';`

**Comment by oozkaya** [comment]
Hello ! I'm in the exact same situation as @kalkih, is there any news about this ? An alternative ?

**Comment by brim-borium** [comment]
Hi @oozkaya I will make a hotfix release today reverting these changes

**Comment by brim-borium** [comment]
> In your file where you use this are you importing the image dimension enum like this ?
> 
> `import 'package:spotify_sdk/enums/image_dimension_enum.dart';`

@oozkaya quick question, did you try this ?

**Comment by brim-borium** [comment]
@oozkaya @kalkih This should be fixed, please test this with the new version 1.0.2.

**Comment by oozkaya** [comment]
Thanks @brim-borium  !

> > In your file where you use this are you importing the image dimension enum like this ?
> > `import 'package:spotify_sdk/enums/image_dimension_enum.dart';`
> 
> @oozkaya quick question, did you try this ?

Well I only used `Spotify.Sdk.getImage` method with only the `imageUri` argument, so I did'nt need ImageDimensions


> @oozkaya @kalkih This should be fixed, please test this with the new version 1.0.2.

Thank you, works perfectly !

**Comment by kalkih** [comment]
Thanks @brim-borium, works great!
